src/Classes/ServiceAPI/MyRadio_Timeslot.php

```diff
@@ -372,9 +372,9 @@ public function toDataSource($mixins = [])
                 'time' => $this->getStartTime(),
                 'start_time' => CoreUtils::happyTime($this->getStartTime()),
                 'duration' => $this->getDuration(),
-                'mixcloud_status' => $this->getMeta('upload_state'),
-                'mixcloud_starttime' => $this->getMeta('upload_starttime'),
-                'mixcloud_endtime' => $this->getMeta('upload_endtime'),
+                'mixcloud_status' => 12,
```

Comment:
this doesn't seem right?

Reply:
ahh FFSS how does that keep sneaking in....